packages/oauth-client/tests/test-EarthdataLoginClient.js

```diff
@@ -1,16 +1,12 @@
-import cryptoRandomString from 'crypto-random-string';
-import nock from 'nock';
-import test from 'ava';
-import { URL, URLSearchParams } from 'url';
+const cryptoRandomString = require('crypto-random-string');
```

Comment:
Note that I updated this test from TypeScript to JS. Partially for consistency, partially because it's a pain to have to recompile the file every time.